.changeset/fast-peaches-approve.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Looks good, let's just get a compilation test, it could just be a wireit script that runs `tsc --target ES5 --noEmit --downlevelIteration` that the `test` script depends on

Reply:
Cool I added a script and wireit entry for `test:es5build` that does that.  Great idea!